packages/ember-runtime/lib/mixins/-proxy.js

```diff
@@ -83,6 +80,10 @@ export default Mixin.create({
   */
   content: null,
 
+  _initializeProxy: on('init', function() {
+    meta(this).setProxy();
```

Comment:
Yea, I was work or less following the pattern with the tag initialization. I will move both of them to `init` so it will be easier to move away (one day) from the eventing stuff.